### DISCUSSION TITLE
Using shell utilities that are coming with python package

### DISCUSSION THREAD

**Discussion by LindezaGrey** [python-poetry/poetry#5426] [Q&A] [answered]
Hi there,

i would like to use the `watchmedo` shell utility function that comes with the `watchdog` package [This one](https://pypi.org/project/watchdog/)

I installed it with:

`poetry add -D watchdog[watchmedo]`

but i can't invoke the command by entering `watchmedo` in the command line.
If i install it with pip, it works:

`python -m pip install -U "watchdog[watchmedo]"`

I already tried to first use `poetry shell` but i am kind of lost here. 
I would appreciate any hints! Thank you

**Comment by clintonroy** [comment, marked as answer]
If I do a =poetry add --dev watchdog[watchmedo]= I see:

`Using version ^2.1.7 for watchdog

Updating dependencies
Resolving dependencies... (0.4s)

Writing lock file

Package operations: 3 installs, 0 updates, 0 removals

  • Installing pyyaml (6.0)
  • Installing python-sat (0.1.7.dev16)
  • Installing watchdog (2.1.7)
`

After which i can run =poetry run watchmedo= and see a help screen.
